### PR TITLE
✨ Add structured meta data

### PR DESCRIPTION
**Pull request by robinvanopstal**
Add `METADATA.yaml` to working group repository in order to provide the following data for #1699 in a structured way:
- Group Name 
- Short Description
- Facilitator
- Members
- Communication Channels

/cc @pbakaus

### CODE REVIEW

**Comment by sebastianbenz** [comment]
@rsimha can you please merge

File: METADATA.yaml [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+facilitator:
+  login: rsimha
+  name: Raghu Simha
+members:
+  - login: danielrozenberg
+    name: Daniel Rozenberg
+  - login: erwinmombay
+    name: Erwin Mombay
+  - login: estherkim
+    name: Esther Kim
```
Comment (rsimha) [inline METADATA.yaml:12]: Could you please add @rcebulko to this list?

**Review by rsimha — CHANGES_REQUESTED** [review]

File: METADATA.yaml (hunk 1 above)
Comment (robinvanopstal) [reply METADATA.yaml:12]: Done! :)

Comment (rcebulko) [reply METADATA.yaml:12]: :tada:

Comment (robinvanopstal) [reply METADATA.yaml:12]: Can you please merge?

**Review by rcebulko — APPROVED** [review]

**Review by rsimha — APPROVED** [review]

**Review by rsimha — COMMENTED** [review]
Our member list has changed since this PR was created. But we can make those in another PR.

**Pull request merged**